changelogs/unreleased/gh-7593-console-debugger.md

```diff
@@ -1,3 +1,3 @@
-## feature/debugger
+## bugfix/debugger
 
-* Introduced readline support to tarantool debugger (gh-7738).
```

Comment:
It doesn't look as a bugfix. Just feature that was added to 2.11 for some reason. Don't know why.

I remember some discussion that we want to allow tooling features in bugfix releases, but I don't remember any result of the discussion or proposals to update the release policy.

I think that the patch shouldn't be there according to our policy. @igormunkin, please, reveal details if you have them.

Reply:
Well, the debugger CLI has an interference with Tarantool's one and this can be considered as a bug. Making proper support of readline in debugger (or vice versa) means a fix. But I would like to hear @igormunkin too.

Reply:
"Ah shit, here we go again." (c)

> It doesn't look as a bugfix. Just feature that was added to 2.11 for some reason. Don't know why.

We discussed this with @kyukhin offline. We both were OK with it.

> I remember some discussion that we want to allow tooling features in bugfix releases, but I don't remember any result of the discussion or proposals to update the release policy.

I also remember us talking regarding the features added for our toolchain (profilers, debugger, gdb extensions, etc). Unfortunately, I don't remember the final decision, but only the fact nobody was against such relaxation of our release policy.

> I think that the patch shouldn't be there according to our policy.

OK, then, feel free to revert it. In fact it means the 2.11 release is totally freezed forever and can be barely used if we talk about our raw tooling.

> Well, the debugger CLI has an interference with Tarantool's one and this can be considered as a bug. Making proper support of readline in debugger (or vice versa) means a fix.

AFAIR, it was covered with the corresponding stub, so it can hardly be considered as a bug.

---

All in all, I have no strong opinion regarding this particular patch. But I still believe that tooling features has to be maintained separately from the *core-related* ones. Otherwise, I see little sense in toolchain enhancements and development. If anybody think the patch shouldn't be landed to this release, feel free to revert it.

Reply:
Ok, I see another tag in changelog - **'tooling'** - that means we update/upgrade/change/revert the programmer's tooling in our platform on our own discretion. @Totktonada do you consider this as a way to go?

Reply:
We (me, @igormunkin, @sergos) agreed to relax the release policy regarding metrics and developers tools later. It is on @igormunkin.

We agreed to add a changelog category next to `feature` and `bugfix`: `tools`.

Reply:
fixed